### PR TITLE
Concatenating the scale to the shortname in order to be more descriptive

### PR DESCRIPTION
**Pull request by mmilitaru**
### Prerequisites

- [ ] I have written a descriptive pull-request title
- [ ] I have verified that there are no overlapping [pull-requests](https://github.com/RHEAGROUP/CDP4-IME-Community-Edition/pulls) open
- [ ] I have verified that I am following the CDP4-IME [code style guidelines](https://raw.githubusercontent.com/RHEAGROUP/CDP4-IME-Community-Edition/master/.github/CONTRIBUTING.md)
- [ ] I have provided test coverage for my change (where applicable)

### Description
Parameter type scale has been concatenated with the short name for display purpose only

### CODE REVIEW

**Comment by alexatrhea** [comment]
Does this solution react correctly to changes in the scale? Like if you change the shortname of the scale in the MeasurementScales browser, does this row react correctly?

File: Requirements/ViewModels/RequirementBrowser/Rows/SimpleParameterValueRowViewModel.cs [hunk 1]
```diff
@@ -82,7 +82,19 @@ public string Name
         /// </summary>
         public string ShortName
         {
-            get { return this.shortName; }
+            get
+            {
+
+                if (this.Scale != null)
+                {
+                    return string.Concat(this.shortName, "(", this.Scale.ShortName, ")");
```
Comment (alexatrhea) [inline Requirements/ViewModels/RequirementBrowser/Rows/SimpleParameterValueRowViewModel.cs]: Should be " [" (note the space) ...... "]".

Comment (mmilitaru) [reply Requirements/ViewModels/RequirementBrowser/Rows/SimpleParameterValueRowViewModel.cs]: It does react on both short name and scale changes

Comment (alexatrhea) [reply Requirements/ViewModels/RequirementBrowser/Rows/SimpleParameterValueRowViewModel.cs]: ok, good, can you still change this line to 

```
return $"{this.shortName} [{this.Scale.ShortName}]";
```

Comment (samatrhea) [reply Requirements/ViewModels/RequirementBrowser/Rows/SimpleParameterValueRowViewModel.cs]: agree with alex

**Pull request merged**